client/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.4.0",
+    "@testing-library/user-event": "^7.2.1",
+    "axios": "^0.19.2",
+    "moment": "^2.24.0",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-moment": "^0.9.7",
+    "react-redux": "^7.1.3",
```

Comment:
Have we decided to use Redux? @Kushagra-0801 @AetherPrior What say? Should we?

Reply:
Well does anyone here know how to use it?

Reply:
You're going to need either Redux or React Context API for managing the app state. Theres stuff to learn both ways, imo.

Reply:
We don't need axios and moment. I am not sure about Redux but I dont think we need it either

Reply:
@VikramjeetD IMHO, our state isn't so extensive that we need another library to help manage it.

Reply:
@Kushagra-0801 
Axios might slightly simplify the code slightly. We could work without it as well though. For our purposes, using Axios and Redux wouldn't hurt though.